lib/public/components/common/selection/infoLoggerButtonGroup/infoLoggerButtonGroup.js

```diff
@@ -11,42 +11,47 @@
  *  or submit itself to any jurisdiction.
  */
 
-import { frontLink } from '../../navigation/frontLink.js';
-import { dropdown } from '../../popover/dropdown.js';
-import { iconCaretBottom } from '/js/src/icons.js';
 import { h } from '/js/src/index.js';
 import { getInfologgerLink } from '../../../../services/externalRouting/getInfologgerLink.js';
 import { CopyToClipboardComponent } from './CopyToClipboardComponent.js';
+import { identifierColumnAndActionsDropdown } from './identifierColumnAndActionsDropdown.js';
+
+/**
+ * Generates an infologger link component with the given infologger link and label.
+ *
+ * @param {string} infologger The infologger link.
+ * @param {string} label The label for the link.
+ * @return {Component|null} The infologger link component, or null if the infologger is not available.
+ */
+const generateInfologgerLink = (infologger, label) => infologger && h('a', { href: infologger, target: '_blank' }, label);
 
 /**
  * Renders a button group that links to a specific details page based on the given parameters,
  * and a dropdown menu with options to copy the link or access the related infologgers.
  *
- * @param {Object} parameters the parameters that specify what links to display for the details page and the infologgers.
- * @return {Component} the infologger button group component
+ * @param {Object} parameters The parameters that specify what links to display for the details page and the infologgers.
+ * @return {Component} The infologger button group component.
  */
 export const infoLoggerButtonGroup = (parameters) => {
     const { flp: flpInfologger, epn: epnInfologger } = getInfologgerLink(parameters);
+    const flpComponent = generateInfologgerLink(flpInfologger, 'Infologger FLP');
+    const epnComponent = generateInfologgerLink(epnInfologger, 'Infologger EPN');
```

Comment:
Here I think that naming can be improved: `flpComponent` do not describe what they actually are, I would keep `flpInfologger` or `flpInfologgerLink`, same for EPN
You can (and should) also rename `getInfologgerLink` to `getInfologgerLinksUrls`